sensor/common/image/service_impl.go

```diff
@@ -78,9 +86,19 @@ func (s *serviceImpl) GetImage(ctx context.Context, req *sensor.GetImageRequest)
 		}, nil
 	}
 
-	img, err := s.localScan.EnrichLocalImage(ctx, s.centralClient, req.GetImage())
+	var err error
+	var img *storage.Image
+	if req.GetImage().GetIsClusterLocal() {
+		img, err = s.localScan.EnrichLocalImage(ctx, s.centralClient, req.GetImage())
+	} else {
+		// ForceLocalImageScanning must be true
+		img, err = s.localScan.EnrichLocalImageInNamespace(ctx, s.centralClient, req.GetImage(), req.GetDeployment().GetNamespace())
+	}
+
 	if err != nil {
-		return nil, errors.Wrap(err, "scanning image via local scanner")
+		err = errors.Wrap(err, "scanning image via local scanner")
+		log.Error(err)
```

Comment:
is this to keep or just debugging?

Reply:
To keep, without it only the admission controller logs would show the error. It was helpful for debugging as well.